plugins/pulp_rpm/plugins/importers/yum/upload.py

```diff
@@ -318,7 +326,8 @@ def _generate_rpm_data(type_id, rpm_filename, user_metadata=None):
     """
 
     # Expected metadata fields:
-    # "vendor", "description", "buildhost", "license", "vendor", "requires", "provides", "relativepath", "filename"
+    # "vendor", "description", "buildhost", "license", "vendor", "requires", "provides",
+    # "relativepath", "filename"
```

Comment:
And this. I'm going to refrain from further comment about the style changes, but please do go through the rest of this PR and revert all the changes that only deal with style.